### ISSUE TITLE
When using vue-eslint-parser, it doesn't parse .vue files

### ISSUE THREAD

**Issue by MystK** [vuejs/eslint-plugin-vue#385]
**Tell us about your environment**

* **ESLint Version:** ^4.4.1
* **eslint-plugin-vue Version:** ^4.2.2
* **Node Version:** v8.9.4

**Please show your full configuration:**
```
module.exports = {
  root: true,
  parserOptions: {
    parser: 'babel-eslint',
    sourceType: 'module'
  },
  env: {
    browser: true,
    node: true
  },
  globals: {
    __static: true
  },
  plugins: [
    'html'
  ],
  "extends": ["plugin:vue/essential"],
  'rules': {
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
    "semi": "error" ,
    "no-nested-ternary": "error",
    "max-len": ["warn", 220],
    "indent": ["error", "tab", { "SwitchCase": 1 }],
    "prefer-template": "warn",
    "quotes": ["warn", "single"],
    "no-delete-var": "error",
    "no-with": "error",
    "space-infix-ops": "error",
    "consistent-return": "error",
    "no-param-reassign": "warn",
    "func-style": ["error", "declaration", { "allowArrowFunctions": true }],
    "require-jsdoc": "error",
    "space-in-parens": "error",
    "default-case": "error",
    "key-spacing": "error",
    "quote-props": ["error", "as-needed"],
    "object-curly-spacing": ["error", "always"],
    "array-bracket-spacing": ["error", "always"],
    "spaced-comment": "error",
    "no-console": "off",
    "require-await": "error",
    "template-curly-spacing": [2, "never"],
    "space-before-blocks": "error",
    "keyword-spacing": "error",
    "no-multiple-empty-lines": [2, {"max": 1}]
  }
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
</template>

<script>
  import _ from 'lodash';
  var crypto = require('crypto');
  const {session} = require('electron')

  import { ipcRenderer } from 'electron';

  const sampleParameters = {
    tenant : 'abacusnextautomation.onmicrosoft.com',
    authorityHostUrl : 'https://login.windows.net',
    clientId : '6667da31-dcd7-4adb-879d-60682dae192a',
    clientSecret : 'iP+eCiQ2rFQfLuz3cgmpiW1A2PaMayO/av2q0iCy4xk='
  };
</script>

<style lang="scss">
</style>
```

**What did you expect to happen?**
When running
````
eslint --ext .vue
````
I expected it to find errors, but it found nothing in the file. When I add:
````
    parser: 'babel-eslint',
````
it works, but it has the error below
````
Use the latest vue-eslint-parser
````
**What actually happened? Please include the actual, raw output from ESLint.**
I expected it to find errors, but it found nothing in the file.

**Comment by mysticatea** [comment]
Please remove HTML plugin:

```
  plugins: [
    'html'
  ],
```

`vue-eslint-parser` parse the script inside of `<script>` as JavaScript. However, HTML plugin strips the `<script>` tag, so `vue-eslint-parser` can't find scripts.

**Comment by MystK** [comment]
Thank you that was the issue!
